zap_dev/lib/src/generator/generator.dart

```diff
@@ -1674,6 +1680,23 @@ class _DartSourceRewriter extends GeneralizingAstVisitor<void> {
     }
   }
 
+  @override
+  void visitInterpolationElement(InterpolationElement node) {
```

Comment:
Since we don't care about `InterpolationString`s, we can override `visitInterpolationExpression` instead.

Reply:
Good catch. Updated.